doc/api-extensions.md

```diff
@@ -2362,3 +2362,7 @@ This API extension indicates that the `/1.0/instances/{name}/uefi-vars` endpoint
 
 This API extension allows newly created VMs to have their `migration.stateful` configuration key automatically set
 through the new server-level configuration key `instances.migration.stateful`. If `migration.stateful` is already set at the profile or instance level then `instances.migration.stateful` is not applied.
+
+## `authorization_apis`
```

Comment:
`_apis` seems redundant in an API extension list. 

How about `authorization_management`?

Reply:
Ahh this is already in main from #12914. I didn't add an API extension for this because there are no changes to the API.

Reply:
@markylaing ah ok we can still change it as its not been released. 
So yes lets use a combined one for both and then release together as intended.

Reply:
Ok sounds good. Shall we just go for `access_management`?

Reply:
https://github.com/canonical/lxd/pull/12987